patches/extra/ungoogled-chromium/add-flag-to-hide-tab-close-buttons.patch

```diff
@@ -1,17 +1,26 @@
---- a/chrome/browser/ui/views/tabs/tab_strip.cc
-+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
-@@ -1909,6 +1909,8 @@ bool TabStrip::SupportsMultipleSelection
- }
+--- a/chrome/browser/ui/views/tabs/tab.cc
++++ b/chrome/browser/ui/views/tabs/tab.cc
+@@ -972,6 +972,8 @@ void Tab::UpdateIconVisibility() {
+   if (IsActive()) {
+     // Close button is shown on active tabs regardless of the size.
+     showing_close_button_ = true;
++    if (base::CommandLine::ForCurrentProcess()->HasSwitch("hide-tab-close-buttons"))
++      showing_close_button_ = false;
+     available_width -= close_button_width;
```

Comment:
`available_width` currently gets decremented here regardless of whether the close button is shown or not, which may cause a small UI regression.